Decl

saliha-sajid opened the pull request.

Commit:
added declations. TODO: Fix AST errors

Commit:
added declations. TODO: Fix AST errors

Commit:
Added declarations. TODO: Fix AST errors

Commit:
Added declarations. TODO: Fix AST errors

Commit:
Fixed AST Node error

Commit:
Fixed AST Node error

oddgfa committed:
Modify gitignore

oddgfa committed:
Fix AST error

ParamDecl already had a String name in its super class.

oddgfa committed:
Merge branch 'Decl' of https://github.com/PavelJurasek/INF5110 into Decl

oddgfa commented:
Tried to do some of that fancy rebasing, but it backfired 🙅‍♂️ The AST error was caused by an extra `name`variable in ParamDecl which were never set and therefore was always null.

oddgfa committed:
Fix AST error

Lots of strange stuff happening with git. But now it should all be back to normal.

oddgfa committed:
Add byte code generation

Byte code is now created from declarations. Add one common `generateCode`. Add possibility to get \"byte type\" from our own Type.

oddgfa commented:
Had some trouble with getting the virtual machine to run from my terminal, so currently it writes the generated commands when running the compiler. Guess this is fine in the development stage.

saliha-sajid commented:
Looks alright this way too.

Saliha

On May 5, 2017 1:14 PM, "Odd Gunnar" <notifications@github.com> wrote:

> Had some trouble with getting the virtual machine to run from my terminal,
> so currently it writes the generated commands when running the compiler.
> Guess this is fine in the development stage.
>
> —
> You are receiving this because you modified the open/close state.
> Reply to this email directly, view it on GitHub
> <https://github.com/PavelJurasek/INF5110/pull/9#issuecomment-299439174>,
> or mute the thread
> <https://github.com/notifications/unsubscribe-auth/AON-agynKZfcexzxuL4v0uErr6Z7hBrhks5r2wSMgaJpZM4NOOSH>
> .
>
